addons/purchase_stock/tests/test_stockvaluation.py

```diff
@@ -1104,11 +1104,11 @@ def _now(*args, **kwargs):
             {'balance': 15.0,   'amount_currency': 30.0,    'account_id': self.stock_input_account.id},
             {'balance': -15.0,  'amount_currency': -30.0,   'account_id': self.company_data['default_account_payable'].id},
         ])
-        self.assertRecordValues(inv.line_ids.full_reconcile_id.reconciled_line_ids, [
+        self.assertRecordValues(inv.line_ids.full_reconcile_id.reconciled_line_ids.sorted('balance'), [
```

Comment:
@Levizar The order is different because when you create the record, the values are put inside the cache but the one2many is not sorted everytime. It means the values inside the cache is not 100% deterministic